content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
 
 ## Feature phase definitions
 
-|            | Alpha      | Beta         | Stable
```

Comment:
Line 14 contains a list of phases, bus is missing Experimental in the list. Might be better to just remove the list of phases.